sdk/cosmosdb/cosmos/package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "@azure/cosmos",
+  "description": "Azure Cosmos DB Service Node.js SDK for SQL API",
+  "keywords": [
+    "cosmosdb",
+    "cosmos db",
+    "documentdb",
+    "document database",
+    "azure",
+    "nosql",
+    "database",
+    "cloud"
+  ],
+  "version": "2.1.5",
+  "author": "Microsoft Corporation",
+  "main": "./lib/src/index.js",
+  "types": "./lib/src/index.d.ts",
+  "engine": {
+    "node": ">=6.0.0"
+  },
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "clean": "rimraf lib",
+    "lint": "tslint --project tsconfig.json",
+    "format": "prettier --write --config .prettierrc.json \"src/**/*.ts\"",
+    "check-format": "prettier --list-different --config .prettierrc.json \"src/**/*.ts\"",
+    "compile": "echo Using TypeScript && tsc --version && tsc --pretty",
+    "compile-prod": "echo Using TypeScript && tsc --version && tsc -p tsconfig.prod.json --pretty",
+    "prepack": "npm install && npm run build",
```

Comment:
Need to remove the prepack script since dependencies are managed via Rush rather than NPM. The build system will explicitly run `build` before `pack`.

Reply:
@bsiegel can you elaborate on this change? Adding this prepack script was the only reliable way I had found to allow installation directly from GitHub. Will rush handle that use case?

Reply:
Thanks for calling out this scenario, it's one I missed in my initial Rush pull request.

I'm communicating with the Rush developer about support for this scenario now. My initial thinking is that we can use the included `common/scripts/install-run-rush.js` script to run `rush install && rush build -t @azure/cosmos` instead of the npm commands above.

Reply:
After doing some more thinking on this, I don't think it's possible to install an individual library within a monorepo like ours directly from GitHub, regardless of whether we use Rush or not. [The syntax](https://docs.npmjs.com/files/package.json#git-urls-as-dependencies) allows you to specify a repo and an optional branch or commit, but not a subdirectory within the repo. @southpolesteve what are your thoughts?

Reply:
Yeah, I don't see a good way around this either. Just a fundamental limitation of monorepo+npm. Is branch-specific or nightly npm packages on the roadmap? Not ideal, but we can work with that.

Reply:
Yep, we do build nightlies (though we don't publish them to NPM), and you can easily trigger a build from any branch on demand.

Reply:
Ok great. Consider this resolved

Reply:
`prepack` removed